test/optimizer/physical_operator_test.cpp

```diff
@@ -580,6 +580,72 @@ TEST(OperatorTests, InnerHashJoinTest) {
   delete expr_b_3;
 }
 
+// NOLINTNEXTLINE
```

Comment:
Instead of just writing a operator test, try to write one in the operator transformer test, so that we are assured it is working correctly.
This is just almost testing the constructors are working correctly

Reply:
done

Reply:
Can you write one please?

Reply:
Thanx (y)